prometheus/desc.go

```diff
@@ -133,7 +149,7 @@ func (v2) NewDesc(fqName, help string, variableLabels ConstrainableLabels, const
 			d.err = fmt.Errorf("%q is not a valid label name for metric %q", label, fqName)
 			return d
 		}
-		labelNames = append(labelNames, "$"+label)
+		labelNames = append(labelNames, label)
```

Comment:
This change is what enables me to drop an extra sort. I can't see any reason variable labels need to be prefixed with a `$` here but there might be something I'm missing.

Reply:
Maybe to mark is a non-constant? But don't remember why it was done - we might want to blame/track why it was added, although if tests are passing... likely fine (: